src/components/shared/DlVirtualScroll/DlVirtualScroll.vue

```diff
@@ -112,6 +115,37 @@ export default defineComponent({
                 : 0
         })
 
+        onMounted(() => {
+            window.addEventListener('load', setItemSize)
```

Comment:
@SorinOpt what is this event youre listening to ? who fires it and what is it used for ?